ingestor/service.go

```diff
@@ -372,6 +378,15 @@ func (s *Service) HandleTransfer(w http.ResponseWriter, r *http.Request) {
 		}
 	}()
 
+	for _, prefix := range s.dropFilePrefixes {
+		if strings.HasPrefix(filename, prefix) {
+			io.Copy(io.Discard, r.Body)
```

Comment:
Do we have a metric that we can incremented to track we dropped them?

Reply:
We now have one per prefix